packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -334,6 +336,56 @@ class _PrivacySettings extends StatelessWidget {
         const UserPreferencesListItemDivider(),
         const _SendAnonymousDataSetting(),
         const UserPreferencesListItemDivider(),
+        const _AdvancedSettings(),
+        const UserPreferencesListItemDivider(),
+      ],
+    );
+  }
+}
+
+class _AdvancedSettings extends StatelessWidget {
+  const _AdvancedSettings({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: <Widget>[
+        Expanded(
+          child: ListTile(
+            onTap: () async {
+              await AppSettings.openAppSettings();
+            },
+            title: Text(
+              appLocalizations.native_app_settings,
+              style: Theme.of(context).textTheme.headlineMedium,
+            ),
+            subtitle: Padding(
+              padding: const EdgeInsets.only(top: SMALL_SPACE),
+              child: Text(
+                appLocalizations.native_app_description,
+                style: const TextStyle(
+                  color: Color.fromRGBO(193, 193, 193, 1.0),
+                  letterSpacing: 0.5,
+                  fontSize: 14,
+                ),
```

Comment:
We have a global text style


`Theme.of(context).textTheme.bodyText2`

(or bodyText1 or something else in there)

Could you use this one, if you need more fine control you can use [.copyWith( ... )](url) behind it

Reply:
This way it's working with dark and light theme and other factors like accessibility

Reply:
Made the changes @M123-dev